### PR TITLE
init.d/functions: Make usage msgs more clear

### PR DESCRIPTION
**Pull request by jamacku**

### CODE REVIEW

**Comment by jamacku** [comment]
CI Output:
```
E: Unable to locate package csbuild

The command "sudo apt-get install -y --force-yes csbuild" failed and exited with 100 during .
```

File: etc/rc.d/init.d/functions [hunk 1]
```diff
@@ -448,7 +448,7 @@ status() {
     if [ "$1" = "-b" ]; then
         if [ -z $pid_file ]; then
             echo $"-b option can be used only with -p"
-            echo $"Usage: status -p pidfile -b binary program"
+            echo $"Usage: status [-p {pidfile} [-b {binary}]] [-l lockfile] {program}"
```
Comment (lnykryn) [inline etc/rc.d/init.d/functions:451]: Looking at the code above, this does not seem correct. Currently we parse it in the -p -l- b order

**Review by lnykryn — CHANGES_REQUESTED** [review]

File: etc/rc.d/init.d/functions (hunk 1 above)
Comment (jamacku) [reply etc/rc.d/init.d/functions:451]: Yes, you are right, thank you.

Comment (jamacku) [reply etc/rc.d/init.d/functions:451]: Actually, I wanted to somehow express this relation:
``"-b option can be used only with -p"``

**Review by lnykryn — APPROVED** [review]

**Pull request merged**